Dockerfile

```diff
@@ -11,7 +11,7 @@
 #
 # Dockerfile defines che-machine-exec production image eclipse/che-machine-exec-dev
 #
-FROM golang:1.10.3-alpine as go_builder
+FROM golang:1.12.8-alpine as go_builder
```

Comment:
1. Did you consider option where we remove vendor folder?
2. If you want to keep vendor folder, should not we add `-mod=vendor` to go build args? See https://github.com/golang/go/wiki/Modules#how-do-i-use-vendoring-with-modules-is-vendoring-going-away
3. Consider simplifying Dockerfile a bit, like `WORKDIR /go/src/github.com/eclipse/che-machine-exec/` now can be just `WORKDIR /che-machine-exec/`